modules/bound-sa-tokens-about.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/bound-service-account-tokens.adoc
+
+[id="bound-sa-tokens-about_{context}"]
+= About bound service account tokens
+
+You can use bound service account tokens to limit the scope of permissions for a given service account token. These tokens are audience and time-bound. This facilitates the authentication of a service account to an IAM role and the generation of temporary credentials mounted to a Pod. You can request bound service account tokens by using volume projection and the TokenRequest API.
+
+[IMPORTANT]
+====
+Because bound tokens are not required until boostrap is complete, they are not supported until the post-bootstrap phase.
```

Comment:
@sttts Can you let me know if the wording in this warning is okay? Per https://github.com/openshift/enhancements/pull/150#discussion_r369101819

Reply:
maybe: Because the cluster install does not make use of bounds tokens, their configuration is done post-install.

Reply:
Thanks, I've updated the wording similar to your suggestion